docker-compose/docker-compose.yml

```diff
@@ -1,8 +1,17 @@
-version: "3"
+# This docker-compose file will spin up an ACS cluster on a local host or on a server and it requires a minimum of 16GB Memory to distribute among containers.
```

Comment:
It says 16 GB, but the sum of all containers is about 8.5 Gb in community version. I would avoid placing any numbers in here as those may change (I know it is a copy form enterprise deployment). If required please add the memory requirements in the docs, for example in the top level README.md. I think it will be better that way